.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -9,4 +9,4 @@ jobs:
             - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
               with:
                   show-progress: ${{ runner.debug == '1' && 'true' || 'false' }}
-            - uses: gradle/wrapper-validation-action@v3
+            - uses: gradle/actions/wrapper-validation@v4
```

Comment:
```suggestion
            - uses: gradle/actions/wrapper-validation@d156388eb19639ec20ade50009f3d199ce1e2808 # v4.1.0
```

Reply:
Applied. I wondered about that, but as the action originally used a `@v...` reference instead of a commit hash, I followed the existing convention.